views/help/forms/form_bug.hbs

```diff
@@ -2,44 +2,41 @@
 <input type="hidden" name="type" value="problem">
 
 <label class="form-group">
-	<label class="control-label">Kurzbeschreibung des Problems</label>
-	<input class="form-control" type="text" name="title" placeholder="Kurzbeschreibung des Problems" required>
+	<label class="control-label">{{$t "help.global.label.briefDescriptionOfTheProblem" }}</label>
+	<input class="form-control" type="text" name="title" placeholder="{{$t "help.placeholder.briefDescriptionOfTheProblem" }}" required>
 </label>
 <label class="form-group">
-	<label class="control-label">Problembeschreibung:</label>
+	<label class="control-label">{{$t "help.form_bug.label.problemDescription" }}</label>
 <textarea class="form-control" cols="50" name="problemDescription" rows="14" required></textarea>
 </label>
 <label class="form-group">
-	<span class="control-label optional">Das Problem tritt auf diesem Gerät auf</span>
+	<span class="control-label optional">{{$t "help.form_bug.label.theProblemOccursOnTheDevice" }}</span>
 	<input type="text" name="device" placeholder="z.B. iPhone X, Samsung Galaxy S10" class="form-control">
 </label>
 <label class="form-group">
 	<span class="control-label">
-		E-Mail-Adresse für eventuelle Rückfragen
+		{{$t "help.form_bug.label.emailAddressForAnyQueries" }}
 	</span>
-	<input class="form-control" type="email" name="replyEmail" placeholder="E-Mail-Adresse" value="{{ userEmail }}" required>
+	<input class="form-control" type="email" name="replyEmail" placeholder="{{$t "help.placeholder.email"}}" value="{{ userEmail }}" required>
 </label>
 <hr>
-<p>Weitere Informationen (optional):</p>
+<p>{{$t "help.global.label.furtherInformation" }}</p>
 <label>
 <div class="row">
 	<div class="col-xs-2">
 		<input type="checkbox" id="consent" name="consent" value="true" checked>
 	</div>
 	<div class="col-xs-10">
 		<span class="control-label" for="consent">
-			Ich bin damit einverstanden,
-			dass zur besseren Bearbeitung meines Problems folgende Informationen mitgeschickt werden dürfen:
-			<li>Browser Name und Version (z.B: Firefox 69.0)</li>
-			<li>Betriebssystem und Version (z.B: Windows 10)</li>
+			{{$t "help.form_bug.text.iAgreeThatTheFollowingInformationMayBe" }}
```

Comment:
```suggestion
			{{{$t "help.form_bug.text.iAgreeThatTheFollowingInformationMayBe" }}}
```
contains HTML -> must not sanatize -> 3 * {

Reply:
done ;)